hrjobcontract/js/src/job-contract/services/contract-pay.service.js

```diff
@@ -63,7 +67,22 @@ define([
 
           deffered.resolve(data || {});
         } else {
-          // TODO call2API
+          console.log('doing hrjobpay');
+          CRM.api3('HRJobPay', 'getoptions', {
+            'sequential': 1,
+            'field': fieldName
+          }).done(function (data) {
+            _.each(data.values, function (option) {
```

Comment:
can we user lodash's `map` to create an array and object generation as suggested above?